The Emplacement Gun

Boomertaters opened the pull request:
emplacement gun go brrrrrrrrrrrrrrrrrr

Boomertaters committed:
Emplacement Gun

Boomertaters committed:
Emplacement Gun Models

Boomertaters committed:
Emplacement Gun Mats

Boomertaters committed:
Emplacement Gun Vgui

Boomertaters committed:
Emplacement Gun Shop Purchase

StrawWagen reviewed (changes requested):
Feels like the emplacement gun should go off to the side with the tau cannon
Should be to the left of the crapvidcam
And 5mb is alot for 1 new texture, should compress that down to <1mb
And compress all the other big textures

Boomertaters committed:
removed nrm from vmt

Boomertaters committed:
Delete materials/models/weapons/tfa_mmod/v_ar3 directory

Boomertaters committed:
Emplacement Gun Mats Fix

Boomertaters committed:
Make weight 800 + move it to near crap vid cam

Boomertaters reviewed (commented):
did the things

StrawWagen commented:
this shit is so fire im cookin with changes to this rn

StrawWagen committed:
POLISH

Style
Change price to skulls, give lots of ammo on secondary purchase
Change clip size to 500
Make the emplacement gun into a minigun
Fires faster the more you hold it down
Add overheating
Literally ignites you if overheating too much

Boomertaters commented:
yay
